Functions.Templates/Templates/HttpTrigger-Python/__init__.py

```diff
@@ -4,21 +4,19 @@
 
 
 def main(req: func.HttpRequest) -> func.HttpResponse:
-    logging.info('Python HTTP trigger function processed a request.')
+    logging.info("Python HTTP trigger function processed a request.")
 
-    name = req.params.get('name')
-    if not name:
-        try:
-            req_body = req.get_json()
-        except ValueError:
-            pass
-        else:
-            name = req_body.get('name')
+    try:
+        # Get value for `name` from parameters or JSON body
+        name = req.params.get("name") or req.get_json().get("name")
+    except ValueError:
+        name = None
 
-    if name:
-        return func.HttpResponse(f"Hello {name}!")
-    else:
-        return func.HttpResponse(
-             "Please pass a name on the query string or in the request body",
-             status_code=400
+    return (

```

Comment:
I personally prefer the more explicit approach that was there rather than the ternary operator since it's so long and you don't realize what's going on until you read all the lines.

Reply:
I trust you on readability and "Pythonicness". I'll revert to the explicit.